src/scss/includes/components/panel.scss

```diff
@@ -36,7 +36,7 @@
   }
 
   &-content {
-    max-height: calc(100vh - 160px);
+    max-height: calc(100vh - 130px);
```

Comment:
Would it be possible to compute this size dynamically ?

Reply:
I think not, except by measuring things in JS.
We'll see that during the incoming homepage redesign